lienzo-core/src/main/java/com/ait/lienzo/client/widget/panel/impl/LienzoPanelHandlerManager.java

```diff
@@ -92,11 +98,23 @@
 
     private boolean m_mouse_button_right = false;
 
+    private IPrimitive<?> m_over_prim = null;
+
     private DragMode m_drag_mode = null;
 
+    private int lastDragMoveX;
+
+    private int lastDragMoveY;
+
+    private MouseEvent lastDragMouseEvent;
+
+    private TouchEvent lastDragTouchEvent;
+
     private IPrimitive<?> m_drag_node = null;
 
-    private IPrimitive<?> m_over_prim = null;
+    private Timer dragLimitsTimer;
```

Comment:
I see the `Timer` being used is from GWT. We just did a big work recently to drop as much gwt stuff as possible. So the idea is to try not to add more GWT stuff, if possible. You think could you please change it to j2cl? (`DomGlobal.setTimeOut`, etc). Thanks!